lib/subtractDays.ts

```diff
@@ -1,15 +1,10 @@
-import { ChronosDate, ensureDate } from './helpers/ensureDate';
+import { ChronosDate } from './helpers/ensureDate';
+import { addDays } from '.';
```

Comment:
Let's use file-based import instead of directory-based. Otherwise linter screams about dep cycles: https://travis-ci.com/github/funbox/chronos/jobs/390796110

Reply:
Updated imports.